### DISCUSSION TITLE
Contain css property

### DISCUSSION THREAD

**Discussion by jamaluddinrumi** [tailwindlabs/tailwindcss#10116] [Ideas]
Is there already Contain css property in tailwindcss?
I've searching for it in tailwind docs but i can't find it.

Contain CSS Property reference: https://developer.mozilla.org/en-US/docs/Web/CSS/contain

**Comment by wongjn** [comment]
Nope! You could use an [arbitrary property](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) or [a plugin to define `contain` utilities](https://tailwindcss.com/docs/plugins#static-utilities) to apply it.

**Reply by jamaluddinrumi** [reply]
ok, thank's mate
